homeassistant/components/fritzbox_callmonitor/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "domain": "fritzbox_callmonitor",
   "name": "AVM FRITZ!Box Call Monitor",
+  "integration_type": "service",
```

Comment:
The device info created for this integration doesn't seem to agree:

![image](https://user-images.githubusercontent.com/195327/198066147-f8c4115e-c617-45c5-a379-c28c3b029fc7.png)

Reply:
Tbh I think `fritzbox_callmonitor` is not as strait forward as others as it's a service of a local device. Thus we do have device information for it.

To put it into context, there are multiple integrations for FritzBox devices
* `fritz` (aka `Fritz!Box Tools`)  which exposes the device itself.
https://www.home-assistant.io/integrations/fritz/
* `fritzbox` (aka `AVM FRITZ!SmartHome`) which exposes the Hub for Fritz!DECT smart home devices.
https://www.home-assistant.io/integrations/fritzbox/
* `fritzbox_callmonitor` which exposes **only a specific service** of the device itself (not included in the `fritz` integration). 
https://www.home-assistant.io/integrations/fritzbox_callmonitor/

Long term it might make sense to integrate the callmonitor into `fritz` but we aren't there yet.